geowebcache/core/src/main/java/org/geowebcache/GeoWebCacheDispatcher.java

```diff
@@ -709,4 +709,14 @@ protected SecurityDispatcher getSecurityDispatcher() {
     public void setSecurityDispatcher(SecurityDispatcher secDispatcher) {
         this.securityDispatcher = secDispatcher;
     }
+
+    /** @return the mainConfiguration */
+    public BaseConfiguration getMainConfiguration() {
+        return mainConfiguration;
+    }
+
+    /** @param mainConfiguration the mainConfiguration to set */
+    public void setMainConfiguration(BaseConfiguration mainConfiguration) {
+        this.mainConfiguration = mainConfiguration;
+    }
```

Comment:
This seems to be defeating the purpose of having Spring do dependency injection.  if the Service objects need the Configuration, they should get it from Spring rather that from another object.